example_test.go

```diff
@@ -0,0 +1,277 @@
+	// Single mapper example.
+	toStringMapper := tupleconv.MakeMapper([]tupleconv.Converter[any, string]{}).
+		WithDefaultConverter(tupleconv.MakeFuncConverter(
+			func(s any) (string, error) {
+				return fmt.Sprintln(s), nil
+			}),
+		)
+	res, err := toStringMapper.Map([]any{1, 2.5, nil})
+	fmt.Println(res, err)
+
+	// Output:
+	// [1
+	//  2.5
+	//  <nil>
+	//] <nil>
+}
+
+func ExampleStringToTTConvFactory() {
+	factory := tupleconv.MakeStringToTTConvFactory().
+		WithDecimalSeparators(",.")
+
+	spaceFmt := []tupleconv.SpaceField{
+		{Type: tupleconv.TypeUnsigned},
+		{Type: tupleconv.TypeDouble, IsNullable: true},
+		{Type: tupleconv.TypeString},
+	}
+
+	converters, _ := tupleconv.MakeTypeToTTConverters[string](factory, spaceFmt)
+	mapper := tupleconv.MakeMapper(converters)
+	result, err := mapper.Map([]string{"1", "-2,2", "some_string"})
+	fmt.Println(result, err)
+
+	// Output:
+	// [1 -2.2 some_string] <nil>
+}
+
+func ExampleStringToTTConvFactory_manualConverters() {
+	factory := tupleconv.MakeStringToTTConvFactory().
+		WithDecimalSeparators(",.")
+
+	fieldTypes := []tupleconv.TypeName{
+		tupleconv.TypeUnsigned,
+		tupleconv.TypeDouble,
+		tupleconv.TypeString,
+	}
+
+	converters := make([]tupleconv.Converter[string, any], 0)
+	for _, typ := range fieldTypes {
+		conv, _ := tupleconv.GetConverterByType[string](factory, typ)
+		converters = append(converters, conv)
+	}
+
+	mapper := tupleconv.MakeMapper(converters)
+	result, err := mapper.Map([]string{"1", "-2,2", "some_string"})
+	fmt.Println(result, err)
+
+	// Output:
+	// [1 -2.2 some_string] <nil>
+}
+
+func ExampleStringToTTConvFactory_convertNullable() {
+	factory := tupleconv.MakeStringToTTConvFactory().
+		WithNullValue("2.5")
+
+	converters, _ := tupleconv.MakeTypeToTTConverters[string](factory, []tupleconv.SpaceField{
+		{Type: tupleconv.TypeDouble, IsNullable: true},
+	})
+	fmt.Println(converters[0].Convert("2.5"))
+
+	// Output:
+	// <nil> <nil>
+}
+
+type customFactory struct {
+	tupleconv.StringToTTConvFactory
+}
+
+func (f *customFactory) MakeTypeToAnyMapper() tupleconv.Converter[string, any] {
+	return tupleconv.MakeFuncConverter(func(s string) (any, error) {
+		return s, nil
+	})
+}
+
+func ExampleTTConvFactory_custom() {
+	facture := &customFactory{}
+	spaceFmt := []tupleconv.SpaceField{{Type: tupleconv.TypeAny}}
+	converters, _ := tupleconv.MakeTypeToTTConverters[string](facture, spaceFmt)
+
+	res, err := converters[0].Convert("12")
+	fmt.Println(res, err)
+
+	// Output:
+	// 12 <nil>
+}
+
+const workDir = "work_dir"
+const server = "127.0.0.1:3014"
+
+func upTarantool() func() {
+	inst, err := test_helpers.StartTarantool(test_helpers.StartOpts{
+		InitScript:   "testdata/config.lua",
+		Listen:       server,
+		WorkDir:      workDir,
+		User:         "test",
+		Pass:         "password",
+		WaitStart:    100 * time.Millisecond,
+		ConnectRetry: 3,
+		RetryTimeout: 500 * time.Millisecond,
+	})
+	if err != nil {
+		fmt.Println(err)
+		return nil
```

Comment:
You could omit check the result for nil before `defer cancel()`.
```suggestion
		return func() {}
```